bugsnag-android-core/src/test/java/com/bugsnag/android/ThreadSerializationTest.kt

```diff
@@ -98,6 +101,7 @@ internal class ThreadSerializationTest {
                 "main-one",
                 ThreadType.ANDROID,
                 false,
+                Thread.State.RUNNABLE,
```

Comment:
It'd be good to have a test case that serializes something other than `RUNNABLE`